Add WS_QWS in the Platforms variable and to define some types

Signed-off-by: Gwenhael Goavec-Merou <gwenhael.goavec-merou@trabucayre.com>
Signed-off-by: Sergey Kostanbaev <sergey.kostanbaev@gmail.com>
---
Index: PyQt-x11-gpl-4.11.3/sip/QtCore/QtCoremod.sip
===================================================================
--- PyQt-x11-gpl-4.11.3.orig/sip/QtCore/QtCoremod.sip
+++ PyQt-x11-gpl-4.11.3/sip/QtCore/QtCoremod.sip
@@ -29,7 +29,7 @@
 
 %Timeline {Qt_4_1_0 Qt_4_1_1 Qt_4_1_2 Qt_4_1_3 Qt_4_2_0 Qt_4_2_1 Qt_4_2_2 Qt_4_2_3 Qt_4_3_0 Qt_4_3_1 Qt_4_3_2 Qt_4_3_3 Qt_4_3_4 Qt_4_4_0 Qt_4_4_1 Qt_4_4_2 Qt_4_5_0 Qt_4_5_1 Qt_4_5_2 Qt_4_5_3 Qt_4_6_0 Qt_4_6_1 Qt_4_6_2 Qt_4_6_3 Qt_4_7_0 Qt_4_7_1 Qt_4_7_2 Qt_4_7_3 Qt_4_7_4 Qt_4_8_0 Qt_4_8_1 Qt_4_8_2 Qt_4_8_3 Qt_4_8_4 Qt_4_8_5 Qt_4_8_6 Qt_5_0_0}
 
-%Platforms {WS_X11 WS_WIN WS_MACX}
+%Platforms {WS_X11 WS_QWS WS_WIN WS_MACX}
 
 %Feature PyQt_Accessibility
 %Feature PyQt_SessionManager
Index: PyQt-x11-gpl-4.11.3/sip/QtCore/qprocess.sip
===================================================================
--- PyQt-x11-gpl-4.11.3.orig/sip/QtCore/qprocess.sip
+++ PyQt-x11-gpl-4.11.3/sip/QtCore/qprocess.sip
@@ -28,7 +28,7 @@
 %If (WS_WIN)
 typedef void *Q_PID;
 %End
-%If (WS_X11 || WS_MACX)
+%If (WS_X11 || WS_QWS || WS_MACX)
 typedef qint64 Q_PID;
 %End
 
Index: PyQt-x11-gpl-4.11.3/sip/QtGui/qwindowdefs.sip
===================================================================
--- PyQt-x11-gpl-4.11.3.orig/sip/QtGui/qwindowdefs.sip
+++ PyQt-x11-gpl-4.11.3/sip/QtGui/qwindowdefs.sip
@@ -27,7 +27,7 @@
 
 typedef QList<QWidget*> QWidgetList;
 %If (- Qt_5_0_0)
-%If (WS_X11)
+%If (WS_X11 || WS_QWS)
 // X11 specific definitions.
 typedef unsigned long WId;
 %End
